cloudwatch-logs-s3-forwarder/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "name_prefix" {
+  description = "A prefix that will be added to resources so that they are unique"
+}
+
+variable "filter_patterns" {
+  description = "List of CloudWatch filter patterns"
+  default     = []
+  type        = "list"
+}
+
+variable "log_group_names" {
+  description = "List of Cloudwatch log group names"
+  default     = []
+  type        = "list"
+}
+
+variable "lambda_handler" {
+  description = "The handler for the Lambda function"
+}
+
+variable "lamda_runtime" {
```

Comment:
Same here, if your handler is written in Java the runtime should be hardcoded and variable not exposed.